portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,70 @@
+         to understand what is heppeneing when apps are opened on phones and help analyze crashes or how much of the phones functions an app uses. For this project we created a way for the screen of the phone to be 
+         recorded along with the insides of the phone to see what the user sees. You can see the website at <a href="https://ui.perfetto.dev/#!/">Perfetto</a>.</p>
+
+      <h3 style="text-align:center;">Photo Gallery</h3>
+      <img src="/images/jon.JPG" alt="Jon" style="width:300px;height:300px;">
+      <img src="/images/jon2.jpg" alt="Jon2" style="width:300px;height:300px;">
+      <img src="/images/jon3.jpg" alt="Jon" style="width:300px;height:300px;">
+      <img src="/images/leaf.jpg" alt="Leaf" style="width:300px;height:300px;">
+      <img src="/images/moon.jpg" alt="Moon" style="width:300px;height:300px;">
+      <img src="/images/boat.JPG" alt="Boat" style="width:300px;height:300px;">
+      
+      <h3 style="text-align:center;">Photography</h3>
+      <p>For the things that I like to photograph, it would just have to be moments with my friends and family. I bought a polaroid camera exacty for the purpose to help keep memeories
+        with my friends. Later on, I want to get all the pictures I took and hang them up on wall just to go through all of the photos and remember what was happening. Another thing that I
+        like about photograpy is that each photo tells its own story. Like that saying "a picture has a thousand words", is very true when it comes to photos.</p>
+      
+      <h3 style="text-align:center;">Hidden Talents</h3>
+      <p>I would not say that I have a lot of hidden talents but one of them is being able to solve a rubiks cube. It took me a while to learn how to do it, I watched tons and tons of videos 
+          of people teaching how to solve the cube. Now I can solve a cube in under 2 minutes.<br><br>Another talent I have is being able to learn from something and develop skills for it very 
+          fast. I have been able to pick up a game play for a few minutes, fail a few times, and be able to be almost great at the game in a short while. I think this trait was developed from 
+          skateboarding. While skating, you fall a lot when you are trying to learn a new trick. You fall and fall and fall. After a while, you want to get up but you keep going regardless and 
+          in the end you end up learning the trick. I think that sense of dedication and of learning the ins and outs of something has helped me to develop this trait.</p>
+      
+
+      <!-- <img src="/images/kota-anything.jpg" alt="Anything Album" style="width:300px;height:300px;">
+      <img src="/images/kota-everything.jpg" alt="Everything Album" style="width:300px;height:300px;">
+      <img src="/images/kota-foto.jpg" alt="FOTO Album" style="width:300px;height:300px;"> -->
```

Comment:
This is your personal repository so it is fine, but usually is a shared repository you should not check in commented out code.

Reply:
+1.